src/lib.js

```diff
@@ -1,4 +1,5 @@
 import * as allMediaQueries from './theme/mediaQueries'
+export * as zIndex from './theme/zIndex'
```

Comment:
`export * as` is not part of the official module spec afaik – you need to import and export again